index.html

```diff
@@ -1,54 +1,183 @@
+  <link rel="stylesheet" href="css/partners.css" />
+  <link rel="stylesheet" href="css/footer.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>Document</title>
 </head>
+
 <body>
-   <header class="header-container">
-       <nav class="navbar">
-           <div class="logo">
-               <a href=""><img src="./images/iDreality.png" alt="IDREALITY"></a>
-           </div>
-           <div class="hamburger">
-             <a href="#"><img class="icon-bar" src="./images/hamburger.png" alt="Icon"></a>
-           </div>
-           <div class="desktop-nav">
-            <ul>
-                <li>Home</li>
-                <li>About</li>
-                <li>Blog</li>
-            </ul>
-           </div>
-       </nav>
-       <div class="header-text">
-           <h3>Lorem ipsum dolor sit amet.</h3>
-           <h1>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dicta!</h1>
-           <div class="border-text">
-               <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. 
-                   Cupiditate magnam delectus vel minima. Dignissimos, magnam!
-                </p>
-           </div>
-           <div class="header-date">
-            <h1>2022.04.11(MON) ~ 13(WED)</h1>
-            <p>@iHub Kenya, Caren Shopping Centre.</p>
-           </div>
-       </div>
-   </header>
-   <section class="program">
-       <div class="program-header">
-            <h2>Main Program</h2>
-            <hr>
-       </div>
-       <div class="program-list">
-           <div class="program-list-items">
-               
-           </div>
-       </div>
-   </section>
+  <header class="header-container">
+    <nav class="social">
+      <ul>
+        <li>
+          <a href="https://twitter.com/odhiambo_ed"><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/edward-odhiambo-6a462a21b/"><i class="fa-brands fa-linkedin"
+             ></i></a>
+        </li>
+        <li>
+          <a href="https://github.com/white3d"><i class="fa-brands fa-github"></i></a>
+        </li>
+        <li><a href="">English</a></li>
+        <li><a href="">French</a></li>
+      </ul>
+    </nav>
+    <nav class="navbar">
+      <div class="logo">
+        <a href="index.html"><img src="./images/logo.png" alt="IDREALITY" /></a>
+      </div>
+      <div class="hamburger">
+        <a href="#"><img class="icon-bar" src="./images/hamburger.png" alt="Icon" /></a>
+      </div>
+      <div class="desktop-nav">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a class="nav-btn" href="#">Share Your Mind</a></li>
```

Comment:
Just after jumping from mobile view to desktop view, this `nav-button` suffers a great "accident"

- [ ] Please ensure that the design of your navigation looks great and responsive from mobile version to desktop view
![image](https://user-images.githubusercontent.com/83596135/157728068-f51ed79b-6a47-47b4-a193-caa266ef5c35.png)